web_ui/src/pages/annotator/components/main-content/labels-shortcuts/labels-shortcuts.component.tsx

```diff
@@ -111,6 +115,21 @@ export const LabelsShortcuts = ({ labels, annotationToolContext, isDisabled = fa
         return isDrawingSmartTool ? smartToolHandler(label) : clickHandler(label);
     };
 
+    const emptyLabel = labels.find((label) => isEmptyLabel(label));
+
+    useHotkeys(
+        hotkeys['empty-label'],
+        (event) => {
+            event.preventDefault();
+
+            if (emptyLabel && selectedAnnotations.length > 0) {
```

Comment:
Why is the check being made on the selected annotations? For detection and segmentation the empty label can also be when the user has no annotations selected - for local annotation projects we always replace the whole scene with a single "full image" annotation that only has the empty label.